### DISCUSSION TITLE
Visual Studio Build issue

### DISCUSSION THREAD

**Discussion by Zaryob** [JSBSim-Team/jsbsim#828] [Support requests]
If you need help or want to ask questions, please use [Discussions](https://github.com/JSBSim-Team/jsbsim/discussions).

This repository's issues are reserved for feature requests and bug reports. Issues that are neither bug reports or feature requests will be transferred to [Discussions](https://github.com/JSBSim-Team/jsbsim/discussions) by the team and the issue will be locked and closed.

***

**I'm submitting a ...**
  - [x] bug report
  - [ ] feature request
  - [ ] support request => **Please do not submit support request here**, see note at the top of this template.

**Describe the issue**

I'm trying to compile in VS2022 I'm getting error

**What is the current behavior?**

```
Build started...
1>------ Build started: Project: prep_plot, Configuration: Release x64 ------
1>cl : command line  warning D9002: ignoring unknown option '/arch:SSE2'
1>easyxml.cxx
1>D:\Development\Adal\jsbsim\src\simgear\xml\easyxml.hxx(27,10): fatal  error C1083: Cannot open include file: 'simgear/compiler.h': No such file or directory
1>plotXMLVisitor.cpp
1>D:\Development\Adal\jsbsim\src\simgear\xml\easyxml.hxx(27,10): fatal  error C1083: Cannot open include file: 'simgear/compiler.h': No such file or directory
1>prep_plot.cpp
1>D:\Development\Adal\jsbsim\src\utilities\prep_plot.cpp(101,10): fatal  error C1083: Cannot open include file: 'input_output/string_utilities.h': No such file or directory
1>Done building project "prep_plot.vcxproj" -- FAILED.
========== Build: 0 succeeded, 1 failed, 0 up-to-date, 0 skipped ==========
========== Elapsed 00:01.112 ==========
```
**What is the expected behavior?**

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**

**Other information**

**Comment by bcoconni** [comment]
Should we infer from the PR #827 that you have resolved this problem ?

**Reply by Zaryob** [reply]
No. Unfortunately.

**Comment by bcoconni** [comment]
The code in `src/utilities` is outdated and can be safely ignored.
